DigitalLearningSolutions.Data.Migrations/202107221605_ChangeEmailMaxLengthOnAdminUsers.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace DigitalLearningSolutions.Data.Migrations
+{
+    using FluentMigrator;
+
+    [Migration(202107221605)]
+    public class ChangeEmailMaxLengthOnAdminUsers : Migration
```

Comment:
Could you please include a screenshot of the before and after Index properties in the db for easy comparison? (you should be able to just right click on the index and choose "properties" in SQL Server Management Studio. The old index can be found on our VM, but make sure to use the non-test db)

Reply:
![image](https://user-images.githubusercontent.com/70278044/126954804-0b402e6d-4e61-4ccb-98e8-eacea7d3fd1b.png)

Reply:
After: 
![image](https://user-images.githubusercontent.com/70278044/126957768-297595f2-4dfa-43c3-8889-fce255c42eff.png)

Reply:
It looks like the "Allow NULLs" tab on the image is cut off - I'm pretty sure NULL values were allowed before, so this change could break the live DB

Reply:
We've also changed varchar to nvarchar - do you mind double checking it's the same as in the Candidates table to make sure these are consistent 🙏

Reply:
Yes nvarchar is consistent with the Candidates table!

Reply:
@Stellake You're right, NULL values were allowed before. I think I've made the right change but I'm having trouble applying the migration to the database so I can see if the properties change correctly, what's the right way to do that?

Reply:
@stellake Got the migrations to apply, and the Index looks correct now!
![image](https://user-images.githubusercontent.com/70278044/127131607-2fe420a6-39f5-4517-85b3-7434a17a90ef.png)